XamarinCommunityToolkit/Behaviors/EffectBehavior.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public class EffectBehavior : BaseBehavior<View>
```

Comment:
Sorry, I cannot realize what's the purpose of this behavior? why I cannot add effect directly?

Reply:
Here is an example from Microsoft where it's used.  I've copied and pasted from this project before to add shadows to a control.

https://github.com/xamarin/xamarin-forms-samples/blob/master/Behaviors/EffectBehavior/EffectsDemo/HomePage.xaml

https://github.com/xamarin/xamarin-forms-samples/blob/master/Behaviors/EffectBehavior/EffectsDemo/Behaviors/EffectBehavior.cs